operators/churner/server.go

```diff
@@ -74,7 +74,11 @@ func (s *Server) Churn(ctx context.Context, req *pb.ChurnRequest) (*pb.ChurnRepl
 		return nil, api.NewResourceExhaustedError(fmt.Sprintf("previous approval not expired, retry in %d", s.latestExpiry-now.Unix()))
 	}
 
-	request := createChurnRequest(req)
+	request, err := createChurnRequest(req)
+	if err != nil {
+		s.metrics.IncrementFailedRequestNum("Churn", FailReasonInvalidRequest)
+		return nil, api.NewInvalidArgError(fmt.Sprintf("invalid request: %s", err.Error()))
```

Comment:
We may not need to prefix it with "invalid request" now with the standard error code, otherwise the end result will look like "InvalidRequest invalid request ...", which is slightly redundant